clickhouse_db_schema/misc.disabled_tests_historical/schema.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE misc.disabled_tests_historical
+(
+    `day` Date,
```

Comment:
Does it make sense to have both `Day` and `timestamp` here?  From what I read, `Day` is the time of the commit while timestamp is the timestamp of the insert, I wonder if we could just use `timestamp` to be the time of the commit and don't need to keep the timestamp of the insert